src/components/Header.js

```diff
@@ -1,10 +1,64 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ user, profileImg, changeUser }) {
+  const handleButtonClick = () => {
+    changeUser();
+  };
+  return (
+    <Wrapper>
+      <button onClick={handleButtonClick}>
+        <img src={profileImg} alt="userProfile"/>
+      </button>
+      <Stack>
+        <h2>{user}</h2>
+        <div>현재 활동중</div>
+      </Stack>
+    </Wrapper>
+  );
 }
+
 const Wrapper = styled.div`
-	background-color: black;
-	color: white;
+  // 상단에 고정
+  position: fixed;
+  top: 0;
+  width: 100%;
+
+  height: 80px;
+  background-color: white;
+  opacity: 0.9;
+
+  display: flex;
+  flex-direction: row;
+
+  button {
+    width: 80px;
+    height: 80px;
+    margin: 0;
+    padding: 5px;
+    margin-left: 15px;
+
+    background-color: white;
+    border: none;
+  }
```

Comment:
styled-components 이용안하신 이유가 있나요?.?👀
혹쉬..` const 변수명=styled.태그타입 ...;` 쓰는게 귀찮아서였나요😂
아래 다 수정제안해드렸구, `UserOn`도 직접 만들어서 적용시켜주세요~!

```suggestion
const Button = styled.button`
    width: 80px;
    height: 80px;
    margin: 0;
    padding: 5px;
    margin-left: 15px;

    background-color: white;
    border: none;
`;
```